modules/services/lorri.nix

```
@@ -60,6 +90,23 @@ in {
 
         Install = { WantedBy = [ "sockets.target" ]; };
       };
+
+      services.lorri-notify = mkIf (cfg.notify) {
```

```suggestion
      systemd.user.services.lorri-notify = mkIf (pkgs.stdenv.hostPlatform.isLinux && cfg.notify) {
```

Until we have a common backend to declare services for Darwin and Linux, the service will have to be gated.